build/charts/antrea/templates/agent/clusterrole.yaml

```diff
@@ -135,6 +135,7 @@ rules:
       - get
       - watch
       - list
+      - update
```

Comment:
can you double check these changes? I don't understand why you are modifying RBAC permissions for the agent, and I don't understand how this change relates to your PR (`update` verb for `antrea-ca` ConfigMap?)

This applies to the change below as well.

Reply:
I've modified it. "antctl set" command will only be supported in FA pod now.